homeassistant/components/vicare/binary_sensor.py

```diff
@@ -75,17 +75,29 @@ class ViCareBinarySensorEntityDescription(
 
 GLOBAL_SENSORS: tuple[ViCareBinarySensorEntityDescription, ...] = (
     ViCareBinarySensorEntityDescription(
-        key=SENSOR_SOLAR_PUMP_ACTIVE,
+        key="solar_pump_active",
         name="Solar pump active",
         device_class=BinarySensorDeviceClass.POWER,
         value_getter=lambda api: api.getSolarPumpActive(),
     ),
     ViCareBinarySensorEntityDescription(
-        key=SENSOR_CHARGING_ACTIVE,
-        name="Domestic Hot Water Charging active",
+        key="charging_active",
+        name="DHW Charging active",
```

Comment:
I renamed this sensor to be in line with the other sensors. "DHW" is just so much shorther than "Domestic Hot Water"

Reply:
Is that a breaking change?

Reply:
I don't think think so. The entity id should ensure that after renaming it's still the same entity